### PR TITLE
[#6957] Wallet shouldn't allows redirect to Home after a transaction …

### LINKED ISSUE #6957: Wallet shouldn't allows redirect to Home after a transaction completion
After a transaction is completed wallet should redirect to the screen that initiated the transaction. Currently it does redirect to the Home page.

This is especially important for the `wallet.settings` hook as current behavior prevents doing anything after the first transaction is completed.

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #6957

### CODE REVIEW

**Comment by goranjovic** [comment]
@flexsurfer @jeluard I'm trying to visualize what is this PR fixing. Can you explain?

File: src/status_im/ui/screens/views.cljs [hunk 1]
```diff
@@ -243,17 +243,44 @@
...
            :wallet-transaction-details   wallet-transactions/transaction-details
-           :login                        login})
+           :login                        login
+           :wallet-settings-hook         wallet-settings/settings-hook})
          {:headerMode       "none"
           :initialRouteName "wallet"})}
+
+       :wallet-send-modal-stack
+       {:screen
+        (nav-reagent/stack-navigator (stack-screens {:wallet-send-transaction-modal
```
Comment (jeluard) [inline src/status_im/ui/screens/views.cljs:253]: What are those?

**Review by jeluard — APPROVED** [review]

**Comment by flexsurfer** [comment]
@goranjovic we want to open modal wallet from the wallet O_o we need this for extensions

**Comment by goranjovic** [comment]
@flexsurfer Ok, I understand that. I don't understand what specifically didn't work with the navigation and how this change fixes it

**Comment by flexsurfer** [comment]
@goranjovic before we had this wallet modal stack only for chat, but if you open it from wallet, it moves you to another stack and when you go back you will return to the wrong screen, but all this navigation thing is confusing for sure

**Review by goranjovic — APPROVED** [review]

**Pull request merged**